### PR TITLE
Resp navbar

### PR DESCRIPTION
**Pull request by SJTGSHIVAM**
- Added Responsive nav bar 
- Added js to handle the event

### CODE REVIEW

**Review by saifudinbadani — APPROVED** [review]
class names are easy to understand looks good to merge.

**Review by romabulani — APPROVED** [review]
Good to merge

**Review by Hnh013 — APPROVED** [review]
the navbar is responsive, and no js errors are coming now , good to merge !

File: molecules/navbar/index.css [hunk 1]
```diff
@@ -7,16 +7,44 @@
...
+}
+
+.tui__nav--list-col {
+  display: grid;
+  box-shadow: var(--shadows-md);
+}
+.tui__nav--menu {
+  display: inline-flex;
+  height: var(--h-2-font-size);
+  width: var(--h-2-font-size);
```
Comment (varnita619) [inline molecules/navbar/index.css:31]: Good use of variables

**Review by varnita619 — APPROVED** [review]
Good naming convention

**Pull request merged**